spec/system/news_article_spec.rb

```diff
@@ -1,5 +1,8 @@
 RSpec.describe "News Article" do
-  let!(:content_item) { content_store_has_example_item("/government/news/christmas-2016-prime-ministers-message", schema: :news_article) }
+  let(:news_article_christmas_2016_path) { "/government/news/christmas-2016-prime-ministers-message" }
+  let(:path) { news_article_christmas_2016_path }
+  let(:example) { :news_article }
```

Comment:
I think it's more confusing to have these values here. For readability I'd assign them within the context they're used in.